site/bot/api/views.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from bot.models import Bot
+from bot.api.serializer import BotSerializer
+
+@api_view(['GET', 'POST'])
+def bot_list(request):
+    """
+    List all code bots, or create a new bot.
+    """
+    if request.method == 'GET':
+        bots = Bot.objects.all()
+        serializer = BotSerializer(bots, many=True)
+        return Response(serializer.data)
+
+    elif request.method == 'POST':
```

Comment:
We are going to create BOT using the django admin, please remove the POST method here